crates/metaslang/bindings/src/lib.rs

```diff
@@ -54,8 +55,55 @@ pub struct Bindings<KT: KindTypes + 'static> {
     context: Option<GraphHandle>,
 }
 
+pub enum FileKind {
+    Unknown,
```

Comment:
I wonder when is it unavoidable to return `Unknown` to the user? can we eliminate this possibility?

Reply:
Hmm... we should be able to map anything that's stored in the stack graph as a `FileDescriptor`, and nodes in the stack graph do not necessarily have a file associated (eg. the `ROOT_NODE`). But since we're returning definitions and references ourselves, we could filter out those.

The other potentially problematic case would be the import path resolver, which now receives and returns a `FileDescriptor`. I initially thought about making it strictly about the "user" namespace, but then I thought it may be too restrictive for some languages. I'm now thinking I may have overreacted... Thoughts?

Reply:
At least for now, there is no need/use case to have users resolve built-in files. I think it is ok to restrict it to "user" namespace if you agree.